bots/common/src/main/java/org/openjdk/skara/bots/common/BotUtils.java

```diff
@@ -42,8 +42,10 @@ public class BotUtils {
      */
     public static Optional<JdkVersion> getVersion(PullRequest pr) {
         var confFile = pr.repository().fileContents(".jcheck/conf", pr.headHash().hex())
-                .orElse(pr.repository().fileContents(".jcheck/conf", pr.targetRef())
-                        .orElseThrow(() -> new RuntimeException("Could not find .jcheck/conf in neither src or target of PR " + pr)));
+                .orElse(pr.repository().fileContents(".jcheck/conf", pr.targetRef()).orElse(null));
+        if(confFile == null){
+            return Optional.empty();
+        }
```

Comment:
```suggestion
        if (confFile == null) {
            return Optional.empty();
        }
```

Reply:
If you fix this formatting, then this is as good as it gets now. I'm not happy with the state of CheckRun, but at least this version should work as we want it without doing excessive extra work.